src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -281,13 +285,54 @@ public void partitionTest() throws Exception {
     }
   }
 
+  private void ingestOMOPTable(
```

Comment:
The naming convention here would be `Omop` not `OMOP`
```suggestion
  private void ingestOmopTable(
```